spec/controllers/api/v2/blocks_controller_spec.rb

```diff
@@ -10,6 +10,38 @@
   let(:result) { Oj.load(response.body).with_indifferent_access[:result] }
   let(:count) { result[:blocks].size }
 
+  context "params transform" do
+    let(:params) do
+      ActionController::Parameters.new({
+        numberFrom: 0,
+        numberTo: 10,
+        transactionFrom: 20,
+        transactionTo: 30,
+        page: 40,
+        perPage: 50,
+        offset: 60,
+        limit: 70
+      })
+    end
+
+    let(:transformed_params) do
+      ActionController::Parameters.new({
+        number_from: 0,
```

Comment:
Can't understand `number_from` and `transaction_from easily please consider renaming them.

Reply:
Rename to `block_number_from` and `min_transaction_count`